lib/util/variable.js

```diff
@@ -24,6 +24,25 @@ function findVariable(variables, name) {
 }
 
 /**
+ * Find and return a particular variable in a list
+ * @param {Array} variables The variables list.
+ * @param {Array} name The name of the variable to search.
+ * @returns {Object} Variable if the variable was found, null if not.
+ */
+function getVariable(variables, name) {
+  var i;
+  var len;
+
+  for (i = 0, len = variables.length; i < len; i++) {
+    if (variables[i].name === name) {
+      return variables[i];
+    }
+  }
+
+  return null;
```

Comment:
I prefer this, so I'll go with it. I was mimicking the `findVariable` function already there.